rgw/v2/lib/resource_op.py

```diff
@@ -215,6 +215,7 @@ def read(self, ssh_con=None):
         self.user_remove = self.doc["config"].get("user_remove", True)
         self.user_type = self.doc["config"].get("user_type")
         self.bucket_count = self.doc["config"].get("bucket_count")
+        self.encryption_keys = self.doc["config"].get("encryption_keys")
```

Comment:
It looks like `encryption_keys` parameter will make confusion in future. Instead of this, can we write `encryption_type` or `encryption_method`

Reply:
@udaysk23 **encryption_keys** is the de facto and taken from the docs https://docs.ceph.com/en/quincy/radosgw/encryption/ 
"The Ceph Object Gateway supports server-side encryption of uploaded objects, with 3 options for the management of encryption keys."

Reply:
Can we make it `encryption_options` or `encryption_option`?

Reply:
It option in the doc